docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -3,15 +3,38 @@
 
 [role="_abstract"]
 
-A devfile describes and defines a cloud native development environment.
+Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development while providing a way for being customized or extended to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating what was previously done manually.
 
-Devfiles are consumed by tools that transform the definition into a cloud workspace composed of multiple cloud native resources. Any number of identical workspaces can be created from the same devfile. They can be shared in various ways including a devfile registry or together with the source code of a project.
+Devfiles are consumed by tools that transform the definition into a cloud workspace composed of multiple containers and cloud-native resources. Also, any number of identical workspaces can be created from the same devfile. This way, workspaces can be used by different teams and shared in various ways, including a devfile registry or together with the source code of a project.
 
+A single devfile can include:
+
+* Guidance about runtime images
+* Examnple code
+* Build and CI commands
+* Deployment options
+
+The benefits of devfiles include:
```

Comment:
If "include" is removed, will the sentence lose some meaning? Can it be seen as noise? Or maybe you wanted some consistency with the previous list (A single devfile can include)?